specs/experimental/alt-da.md

```diff
@@ -274,22 +274,22 @@ The input data stored on the DA storage for the given `<commitment>`.
 The status of the challenge for the given `<commitment>` at the given `<blocknumber>` on the L1
 DataAvailabilityChallenge contract.
 
-[faultproofs]: ../fault-proof/index.md
+[faultproofs]: ./fault-proof/index.md
```

Comment:
why this diff?

Reply:
this was another thing I missed due to pulling in changes before pushing -- FP moved *out* of the experimental folder, and when I pulled the change in, it didn't apply to `alt-da.md`. Fixed in latest version